### ISSUE TITLE
[ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning

### ISSUE THREAD

**Issue by jrobinsonc** [doasync/eslint-config-airbnb-standard#4]
I'm seeing this warning:

```shell
[ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "standard")
```

It can be fixed upgrading `eslint-config-standard` from v10 to v12.

More info about the issue:  
<https://github.com/standard/eslint-config-standard/issues/127>

**Comment by doasync** [comment]
Fixed in v3.0.0
